javascripts/main.js

```diff
@@ -0,0 +1,82 @@
+
+
+function carsToDOM(_car_inventory){
+
+	_car_inventory.map(function(currentCar){
+		console.log(currentCar);
+		document.getElementById("card").innerHTML +=
+		`<div class="prodCard col-md-4">
+		<h2>${currentCar.year}</h2>
+		<h2>${currentCar.make}</h2>
+		<h2>${currentCar.model}</h2>
+		<p id="desc">${currentCar.description}</p>
+		<h2>${currentCar.price}</h2>
+		</div>`
+		if ((cars.length %3) === 0 ){
+			document.getElementById("card").innerHTML +=`
+			</div>
+			<div class="row">`
+		}
+	})
+	listeners();
+}
+inventory.loadInventory(carsToDOM);
+
+var cars = document.getElementsByClassName("prodCard");
+// console.log(cars);
+function listeners() {
+	for (i=0; i<cars.length; i++){
+		cars[i].addEventListener("click", function(){
+			for (var x= 0; x<cars.length; x++) {
+				cars[x].classList.remove("focusBorder");
+			}
+			this.classList.toggle("focusBorder");
+			input.focus();
+			input.value = "";
+		})
+	};
+}
+
+function changeDescription(event) {
+	var thingToChange = document.getElementsByClassName("focusBorder")[0];
+	if (thingToChange) {
+		var newDesc = input.value;
+		var descChange = thingToChange.getElementsByTagName("p")[0];
+		descChange.innerHTML = newDesc;
+	}
+	if(event.code === "Enter") {
+		input.value = "";
+	}
+
+}
+
+input.addEventListener("keyup", changeDescription);
+
+// addRows();
+// Loop over your array of cars and build up an HTML string to build a card for each car.
```

Comment:
When you push code to "production", you always want to make sure that everything is consistently formatted and that all console logs and non-essential comments have been trimmed out of your code. 

Before you merge, make sure you take care of these things.